src/components/Dashboard/Balance.js

```diff
@@ -47,7 +47,7 @@ const Balance = React.memo(function Balance({
         `}
       >
         {loading ? (
-          <Loading height={86} />
+          <Loading height={150} />
```

Comment:
I think @fabriziovigevani changed this recently because it was off balance, wanted to ping to re-confirm.